roman_numerals/kotlin/README.md

```diff
@@ -0,0 +1,9 @@
+## Prerequisite
+
+- Java Development Kit >= 8 (see [AdoptOpenJDK](https://adoptopenjdk.net) for example)
+
+## How to start?
+
+1- Run tests with `make`
+
+2- Hack `App.kt` in `src/main/kotlin`
```

Comment:
De mon côté je lance via docker pour ne pas avoir à installer le jdk : 
```bash
docker run -it -v <absolute_path_to_sogilis-katalog>:/home openjdk:8-jdk
```
puis
```bash
apt update && apt install make
```
et enfin
```bash
cd home/roman_numerals/kotlin/ && make
```

Reply:
(Peut être que cette procédure peut être rajoutée à la doc)

Reply:
Why not.
Done.